plugin-k8s-vpa/pkg/plugin/plugin.go

```diff
@@ -283,7 +283,7 @@ func (a *Autoscaler) VerticalRecommendation(now int64) ([]*proto.RecommendedPodR
 	//	})
 	//}
 	//TODO remove this code after fulfilling https://github.com/pivotal/skenario/issues/100
-	vpa := a.recommender.GetClusterState().Vpas[model.VpaID{Namespace: "", VpaName: ""}]
+	vpa := a.recommender.GetClusterState().Vpas[model.VpaID{Namespace: "", VpaName: "my-app-vpa"}] //TODO fix hardcoded vpa name in an issue https://github.com/pivotal/skenario/issues/114
```

Comment:
Tip: You can just say `// TODO(#114): fix hardcoded vpa name`.

Reply:
Fixed it